spec/requests/sales_logs_controller_spec.rb

```diff
@@ -559,6 +559,26 @@
           sign_in user
         end
 
+        it "does not have organisation values" do
+          get "/sales-logs", headers: headers, params: {}
+          expect(page).not_to have_content("Owned by")
+          expect(page).not_to have_content("Managed by")
+        end
+
+        context "and organisation has absorbed organisations" do
+          let(:merged_organisation) { FactoryBot.create(:organisation) }
+
+          before do
+            merged_organisation.update!(absorbing_organisation: organisation, merge_date: Time.zone.yesterday)
+          end
+
+          it "has organisation values" do
```

Comment:
can we name these tests consistently between sales and lettings if they're testing the same thing so they're easier to find in future?

Reply:
Sure!